packages/modul-components/src/components/phonefield/phonefield.ts

```diff
@@ -105,7 +112,36 @@ export class MPhonefield extends ModulVue {
     emitContrySelected(country: Country): void { }
 
     get isoCountries(): string[] {
-        return this.countries.map(contry => contry.iso2);
+        return this.countriesSorted.map(contry => contry.iso2);
+    }
+
+    get countriesSorted(): CountryOptions[] {
+        let finalCountriesListe: CountryOptions[] = [];
+        const separatorCountry: CountryOptions[] = this.priorityIsoCountries.length > 0 ? [{
+            name: '',
+            iso2: '',
+            dialCode: ''
+        }] : [];
+
+        const priorityCountries: CountryOptions[] = [];
+        this.priorityIsoCountries.forEach((isoPriorityCountry: string) => {
+            const currentCountry: CountryOptions | undefined = this.countries.find((isoCountry: CountryOptions) => isoCountry.iso2 === isoPriorityCountry);
+            if (currentCountry) {
+                priorityCountries.push(currentCountry);
+            }
+        });
+
+        const allOtherCountries: CountryOptions[] = this.countries.filter((country: CountryOptions) => {
```

Comment:
ici aussi, je pense que tu peux faire finalCoutriesList.push(this.countries.filter(...);